home.php

```diff
@@ -6,6 +6,13 @@
 $pdo->setAttribute(PDO::ATTR_ERRMODE, PDO::ERRMODE_EXCEPTION);
 $user_id = $_SESSION['user_id'];
 
+/* Abir */
+$loggedInUser = "NULL";
+if($_SESSION["user"] != "NULL") {
```

Comment:
Use `isset()` to check this condition. Note that the session variable already has a key called 'user_id', it could/should be used in this case.